language/Spanish/strings.po

```diff
@@ -1095,15 +1106,18 @@ msgid ""
 "Please select a different machine.\n"
 "\n"
 msgstr ""
+"Falta la máquina seleccionada o una o más de sus ROMs o imágenes CHD. "
+"Por favor, elige otra máquina.\n"
+"\n"
 
 #: src/frontend/mame/ui/selgame.cpp:1737 src/frontend/mame/ui/selsoft.cpp:630
 msgid "Press any key to continue."
-msgstr ""
+msgstr "Pulsa cualquier tecla (menos ESC) para continuar."
```

Comment:
Adding "menos ESC" to this string is misleading.  Firstly, UI Cancel can be remapped, so assuming it's mapped to the escape key is not acceptable.  Secondly, this string is used in the missing software popup which can be dismissed with the UI Cancel key (escape by default), so it isn't even correct with default mappings.